### PR TITLE
<Description> : Adding isTomorrow and isYesterday extensions to Date.

### PR DESCRIPTION
**Pull request by Khalian**
## Checklist

- [x] New Extension
- [x] New Test
- [x] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by lfarah** [comment]
I'd also add an ```isThisWeek()``` and ```isThisMonth``` because it is really good in apps that show a calendar separated by overdue, today, this week and this month.

What do you think?

**Comment by Khalian** [comment]
Yea sure, I can add those.

**Comment by Khalian** [comment]
Not sure how to do isThisWeek

File: Sources/DateExtensions.swift [hunk 1]
```diff
@@ -116,6 +116,27 @@ extension Date {
...
+    public var isTomorrow: Bool {
+        let format = DateFormatter()
+        format.dateFormat = "yyyy-MM-dd"
+        let tomorrow = Calendar.current.date(byAdding: .day, value: 1, to: Date())
+        return format.string(from: self) == format.string(from: tomorrow!)
+    }
+    
+    /// EZSE: Check date if it is within this month.
+    public var isThisMonth: Bool {
+        return self.month == Date().month
```
Comment (furuyan) [inline Sources/DateExtensions.swift]: Maybe, `isThisMonth` should compare each year as well.

**Comment by lfarah** [comment]
Hey @Khalian, sorry for the long wait:
``` swift
extension Date {
    
    func isThisWeek() -> Bool {
        return self.minutesInBetweenDate(Date()) <= 24 * 60 * 7
    }
}
```

**Comment by furuyan** [comment]
IMHO, this method is better to compare date  with firstDayOfThisWeek as follows.
```
var firstDayOfThisWeek: Date {
    let date = Date()
    let calendar = Calendar.current
    var components = (calendar as NSCalendar).components([.year, .month, .day, .hour, .minute, .second], from: date)
    components.weekday = 0
    components.hour = 0
    components.minute = 0
    components.second = 0
    return calendar.date(from: components)!
}

var isThisWeek: Bool {
    let oneWeekMinutes: TimeInterval = 24*60*60*7
    return self.minutesInBetweenDate(firstDayOfThisWeek) < oneWeekMinutes
}
```

**Comment by Khalian** [comment]
Done

**Comment by Khalian** [comment]
Hey guys, would you mind approving this ?

**Pull request merged**